harmonisation/specimen/taxonomicTerms.md

```diff
@@ -0,0 +1,62 @@
+# Taxonomic terms
+
+This is a set of terms which are combined in this single file.
+We combined these terms as we use a similar approach to harmonised them.
+Additionally, it would unnecessarily grow the amount of terms files in this repository.
+
+## openDS
+These terms will be harmonised for taxonomy.
+- `dwc:class`
+- `dwc:family`
+- `dwc:genus`
+- `dwc:infraspecificEpithet`
+- `dwc:kingdom`
+- `dwc:order`
+- `dwc:phylum`
+- `dwc:scientificNameAuthorship`
+- `dwc:specificEpithet`
+- `dwc:taxonRank`
+- `dwc:typeStatus`
+
```

Comment:
Does this include geology taxonomy? Maybe add a note about how that's still a work in progress.

Reply:
Yes this includes geology. However, I expect that this will be empty for geological specimen. Is there such a thing as geology taxonomy. To be fair, most geological specimen I have seen are called "Rock"

Reply:
yes, for geologic speciments, rock type would be the obvious taxonomy. You can see a listing of some classifications with notes here https://github.com/smrgeoinfo/vocabulary/tree/main/lithology